modules/light-clients/08-wasm/types/vm.go

```diff
@@ -4,14 +4,19 @@ import (
 	cosmwasm "github.com/CosmWasm/wasmvm"
 	wasmvmtypes "github.com/CosmWasm/wasmvm/types"
 
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
 	"github.com/cosmos/ibc-go/v7/modules/core/exported"
 )
 
 var (
-	WasmVM        *cosmwasm.VM
+	WasmVM *cosmwasm.VM
+	// Store key for 08-wasm module, so that the KV store can be retrieve
+	// in ClientState Initialize function to check the code hash of the
+	// contract and determine if the light client is allowed to be instantiated.
```

Comment:
I'd add slightly more context

```suggestion
	// Store key for 08-wasm module, required as a global so that the KV store can be retrieved
	// in the ClientState Initialize function which doesn't have access to the keeper. 
	// The storeKey is used to check the code hash of the contract and determine if the light client
	// is allowed to be instantiated.
```